packages/ra-ui-materialui/src/input/SearchInput.spec.tsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import { render } from '@testing-library/react';
+import { testDataProvider } from 'ra-core';
+
+import { AdminContext } from '../AdminContext';
+import { SearchInput } from '.';
+import { FilterForm } from '../list';
+
+describe('<SearchInput />', () => {
+    it('should not render label if passed explicit `undefined` value', async () => {
```

Comment:
I don't understand this test.

Reply:
Added to cover the case of passing explicit `undefined` for the SearchInput's label property (more info [here](https://github.com/marmelab/react-admin/pull/8439#discussion_r1033939044))